.github/workflows/cxx.yml

```diff
@@ -1,83 +1,97 @@
-# Copyright © 2024 Bret Brown
-# SPDX-License-Identifier: MIT
```

Comment:
why do you want to remove these lines. I think we should keep them.

btw, @camio , should we add a mention for YAML files into https://github.com/beman-project/beman/blob/main/docs/beman-standard.md#file-contents?

Reply:
> btw, @camio , should we add a mention for YAML files into https://github.com/beman-project/beman/blob/main/docs/beman-standard.md#file-contents?

Sounds like a promising idea. Would you be willing to create a PR or issue for it?

Reply:
Yes. I will do it.

Reply:
Merged https://github.com/beman-project/beman/pull/44